### PR TITLE
Issue 944 cleanup

### PR DESCRIPTION
**Pull request by yanokwa**
Builds on #948. Please merge that first!

Start reviewing the changes at e0bd7ab59101ca295f6079f37b005910d5f89dc8

#### What is included in this PR?
* Tried to put all the legacy stuff in one section that matched the non-legacy stuff
* Tried to reduce the number of sections and ToC levels we have
* Tried to tighten up the language
* Removed the GAE backup section since it's deprecated
* Returned the publish data summary because I think it's nice to have
* Removed the blurb about open-source because it's a little heavy handed and out of place

#### What is left to be done in the addressed issue?
* File an issue to add the aggregate-cli tool to the docs.

### CODE REVIEW

**Comment by yanokwa** [comment]
@ggalmazor I've approved https://github.com/opendatakit/docs/pull/948. If you think this PR is an improvement, then approve and merge both. If you'd like changes, leave comments or send in another PR!

File: odk1-src/aggregate-deployment-planning.rst [hunk 1]
```diff
@@ -85,23 +79,23 @@ Ongoing support
 Infrastructure architecture
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-In these docs we explain how to deploy a monolithic server with everything Aggregate needs to run (Tomcat, PostgreSQL, SSL support), but there's a multitude of different scenarios that you might want to consider:
+In these docs we explain how to deploy a monolithic server with everything Aggregate needs to run (Tomcat, PostgreSQL, SSL support), but there are a multitude of scenarios that you might want to consider:
```
Comment (ggalmazor) [inline odk1-src/aggregate-deployment-planning.rst]: I'm not a native English speaker, but I'm curious: is it not "there is a multitude of scenarios" (one multitude) as in "there is a pack of cigarettes" (one pack)?

File: odk1-src/aggregate-backup.rst [hunk 2]
```diff
@@ -62,6 +62,8 @@ Restore
 
         psql -f [ backup file location ] [ dbname ]
 
+3. Start Tomcat.
```
Comment (ggalmazor) [inline odk1-src/aggregate-backup.rst]: Nice catch, thanks!

File: odk1-src/aggregate-do.rst [hunk 3]
```diff
@@ -1,117 +1,78 @@
...
+    .. image:: /img/aggregate-do/user-data.*
+      :alt: Inserting Cloud-Config script under User Data section
 
-- Check your provider's instructions if your domain is not hosted or managed by DigitalOcean
 
-.. note::
+8.  In the :guilabel:`Choose a hostname section`, enter the domain name (e.g., your.domain). This hostname will be used by the Cloud-Config script to configure your server’s domain name. You must enter the same domain name to enable HTTPS.
 
-  - If you don't own a domain, services such as `FreeDNS <https://freedns.afraid.org>`_ offer creating subdomains under a range of domains for free.
+9.  You will not need to add public SSH keys (unless you know what that is and you want to).
```
Comment (ggalmazor) [inline odk1-src/aggregate-do.rst]: But then they won't be able to log into the machine and run the cerbot commands, update Aggregate...

Comment (ggalmazor) [reply odk1-src/aggregate-do.rst]: OK, I see now that DO sends you the root password

**Pull request merged**